docs-source/spring/content/development/cli/_index.md

```diff
@@ -957,6 +1011,208 @@ GraalVM Compile Commands
             E-mail: obaas_ww@oracle.com
     ```
 
+### User Management
+
+Manage users let you store an unlimited amount of users and apply the access permission using roles into the platform. A user role represents a specific set of tasks or responsibilities assigned to a group of users within the platform. Assigning user roles enables organizations to control the actions each user can perform in the system, streamlining their workflow.
+
+**User Roles**
+
+* ROLE_ADMIN: Users with this role usually have complete access rights, allowing them to manage all aspects of the platform, such as adding new users, modifying settings, viewing and editing all content in the system.
+
+* ROLE_USER: This role might be able to connect with Admin Service, create and list for applications (namespaces); use bind command; deploy, list and scale workloads (services).
+
```

Comment:
Missing ROLE_CONFIG_EDITOR role. Can be documented as currently unused in the environment

Reply:
remove "might".
don't use semi-colons in lists, use commas.
add a comma after the and

Reply:
"the" Admin Service